corehq/apps/reports/formdetails/readable.py

```diff
@@ -252,6 +252,13 @@ def add_property_error(self, case_type, case_property, form_id, message):
         return prop
 
     def add_property_detail(self, detail_type, root_case_type, module_id, column):
+        if column.field == '#owner_name':
+            return None
+
+        parts = column.field.split('/')
+        if parts and parts[0] == 'user':
```

Comment:
Does this mean usercase properties in case details will no longer show on case summary?

Reply:
They never showed before (as properties that aren't loaded into forms weren't shown in the case summary page - bug #1 in the description). 
User cases don't show as a separate case type (although I'm not sure why), but these would ideally show up there if that case type showed in the summary. Right now it doesn't make much sense for this property to show as a property of a different case type as I think that's confusing. 

Also, the syntax we use to define these in case lists / details is problematic - if you have an case index named `user`, you would never be able to show those cases in a case detail because the index syntax is overloaded here. That's a different bug though.